packages/jmix-react-antd/src/ui/hotkeys/HotkeyInfo.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { Descriptions } from "antd";
+import { observer } from "mobx-react";
+import { useIntl } from "react-intl";
+import { HotkeyConfig } from "@haulmont/jmix-react-web";
+
+function groupBy<T extends Record<string, any>>(arr: T[], propName: string): Record<string, T[]> {
+  return arr.reduce<any>((acc, el) => {
+    if (acc[el[propName]]) {
+      acc[el[propName]].push(el);
+    } else {
+      acc[el[propName]] = []
+      acc[el[propName]].push(el);
+    }
+    return acc;
+  }, {});
+}
+
+interface HotkeyInfoProps {
+  hotkeyConfigs: HotkeyConfig[];
+}
+export const HotkeyInfo = observer(({hotkeyConfigs}: HotkeyInfoProps) => {
```

Comment:
This is how the component looks when there is a lot of hotkeys defined:

![image](https://user-images.githubusercontent.com/8298981/140961950-3d7cd970-cf32-49d8-8adb-8ad3dd0f6be5.png)

I think it's better to add a scroll area inside the modal.

Reply:
Now it looks like this:
![554ed640786d23ffdd66256da7e8ef4a](https://user-images.githubusercontent.com/34520941/141467228-cc98826c-a53d-4078-8f38-01188727b2f7.png)